src/compat.ts

```diff
@@ -150,3 +152,10 @@ export function playstrategyVariants(
       return rules;
   }
 }
+
+export const amazonsChessGroundFen = (fen: string): string => {
```

Comment:
Can this be renamed to `Chessground`? we dont refer to it as `ChessGround` anywhere. Think this is then used in lila